CMakeLists.txt

```diff
@@ -2,7 +2,9 @@
 cmake_minimum_required( VERSION 2.8 )
 project( xrdcl-authz-plugin )
 
-find_package( XRootD REQUIRED CONFIG)
+set( CMAKE_MODULE_PATH /usr/share/xrootd/cmake )
```

Comment:
I didn't need this in my container.  Can you try without?  The default paths should include this.